README.md

```diff
@@ -15,16 +16,20 @@ The following are required to install competitions:
   * RVM installed on server
   * Capistrano for deployment
 
+
 # Compatibility
 
   * Ruby:   2.6.6
   * Rails:  5.2.4
 
+
 # Shibboleth IDP
+
   You will need an IDP for your staging and production instances which may be coordinated with your university or the NIH.
 
   [One Login](https://www.onelogin.com/developer-signup) can be used to set up an IDP for development.
```

Comment:
Maybe there needs to be a user section and include something about registered users